src/Modules/OrchardCore.Commerce/Views/Payment/Index.cshtml

```diff
@@ -0,0 +1,98 @@
+@using Money
+@using OrchardCore.Commerce.Abstractions
+@using OrchardCore.Commerce.Extensions
+@using OrchardCore.Commerce.Models
+@using OrchardCore.Entities
+@using OrchardCore.Settings
+
+@inject IShoppingCartPersistence ShoppingCartPersistence
+@inject IPriceService PriceService
+@inject IPriceSelectionStrategy PriceSelectionStrategy
+@inject ISiteService SiteService
+
+@{
+    // Shopping cart ID is null by default currently.
+    var currentShoppingCart = await ShoppingCartPersistence.RetrieveAsync();
+
+    var cartIsNotEmpty = currentShoppingCart.Count != 0;
+
+
+    Amount defaultTotal = new();
+    string publishableKey = string.Empty;
+
+    if (cartIsNotEmpty)
+    {
+        var totals = await currentShoppingCart.CalculateTotalsAsync(PriceService, PriceSelectionStrategy);
+
+        // We can only pay in one currency at a time. Later we have to figure out what to do if there are multiple
+        // totals i.e., multiple currencies. https://github.com/OrchardCMS/OrchardCore.Commerce/issues/132
+        defaultTotal = totals.SingleOrDefault();
+
+        publishableKey = (await SiteService.GetSiteSettingsAsync()).As<StripeApiSettings>().PublishableKey;
+    }
+}
+
+<zone name="Header">
+    <header class="masthead">
+        <div class="container">
+            <div class="intro-text">
+                <div class="intro-lead-in"></div>
+                <div class="intro-heading text-uppercase">@T["Checkout"]</div>
+            </div>
+        </div>
+    </header>
+</zone>
+<section id="checkout">
+    @if (cartIsNotEmpty)
+    {
+        <div class="row">
+            <div class="col-12 mx-auto">
+                <h1>@T["Orchard Core Commerce"]</h1>
+                <h2>@T["Stripe Payment"]</h2>
+            </div>
+        </div>
+        <div class="card-payment-form_containter">
+            <form id="card-payment-form" class="card-payment-form">
+                <fieldset>
+                    <div class="row">
+                        <label for="card-payment-form_name">@T["Name"]</label>
+                        <input id="card-payment-form_name" type="text" required>
+                    </div>
+                    <div class="row">
+                        <label for="card-payment-form_email">@T["Email"]</label>
+                        <input id="card-payment-form_email" type="email" required>
+                    </div>
+                    <div class="row">
+                        <label for="card-payment-form_phone">@T["Phone"]</label>
+                        <input id="card-payment-form_phone" type="tel" required>
+                    </div>
+                </fieldset>
+                <fieldset>
+                    <div class="row credit-card">
+                        <div id="card-payment-form_card"></div>
```

Comment:
If you forget to fill the `publishableKey` then you get this which is particularly unhelpful:
![image](https://user-images.githubusercontent.com/4007293/182024481-f2bd7621-8c2f-4c29-9020-196103f500c5.png)

Let's add some placeholder text instead. If it's set up, the `<script>` block below will override it.
```suggestion
                        <div id="card-payment-form_card">@T["Stripe payment is not configured for this site."]</div>
```